bigquery_etl/glean_usage/templates/baseline_clients_first_seen.metadata.yaml

```diff
@@ -3,7 +3,11 @@
 
 friendly_name: Baseline Clients First Seen
 description: |-
-  Captures the earliest server date that we observe a particular client.
+  Captures the earliest server date that we observe a particular client in the
+  baseline table. A client may have a `first_seen_date` that is earlier than the
+  epoch of the baseline table; this is the result of a migration from a legacy
+  data collection system. The `first_seen_date` and the `is_new_profile` fields
+  therefore have slightly different semantics.
```

Comment:
We should check with @gkabbz about terminology here. My understanding is that there really is no concept of a "profile" for mobile apps, although we tend to talk about "new profiles" for symmetry with desktop. is_new_profile is probably the most clear choice, even though one could argue it's technically incorrect.

Reply:
I don't have strong feelings about referring about it as new profiles. If it helps with symmetry with desktop I vote for it. Helps reduce the learning curve when switching between products. In desktop a profile = client_id which would hold here too right?

Reply:
I'm also using the terminology from the [cross ping mobile ETL](https://docs.google.com/document/d/1aayc2G0qFJP7EOSvELgUsYklpyixeZChVMYtHlbCpw8/edit#heading=h.9zc3l8c1meuo) which uses `is_new_profile`.

Reply:
> In desktop a profile = client_id which would hold here too right

Correct.

Let's keep `is_new_profile` as the name.